app/controllers/drops_controller.rb

```diff
@@ -73,11 +73,11 @@ def set_drop
     end
 
     def drop_params
-      params.require(:drop).permit(:sc_track, :title)
+      params.require(:drop).permit(:sc_track, :title, :latitude, :longitude)
     end
 
     def place_params
-      params.require(:drop).require(:place).permit(:name, :longitude, :latitude, :location)
+      params.require(:drop).permit(place: [:name, :longitude, :latitude, :location]).fetch(:place, {})
```

Comment:
Can you explain this a bit @carpodaster? It's new to me. From what I can tell it allows one to associate a place w/ a drop (by changing `require` to `permit`, right? But what does the `fetch(:place, {})` thing do?